### ISSUE TITLE
Native integration with Aurora

### ISSUE THREAD

**Issue by Cheerpipe** [Cheerpipe/RGBFusionCli#8]
RGBFusionCli is stable enough for bein integrated into aurora as a wrapper just like new Asus Aura integration.

I don't know how to do this in Aurora so this will take some time at least for me alone. Would be nice to get some help with this.

**Comment by Hukuma1** [comment]
Did you already try posting this on Aurora's Github?

**Comment by Cheerpipe** [comment]
No, I have to learn how to write a Aurora device without break Aurora or wait for help.

**Comment by Cheerpipe** [comment]
Hope this could be merged some day

https://github.com/antonpup/Aurora/pull/2057 (rewrite branch).
